### ISSUE TITLE
TypeScript error in featureDetection.ts

### ISSUE THREAD

**Issue by JReinhold** [jaredpalmer/react-fns#53]
When trying to build the project, I get a TypeScript error in `src/utils/featureDetection.ts`:

```javascript
9   (window as EventTarget).addEventListener('testPassive', null, opts);
                                                            ~~~~

src/utils/featureDetection.ts(9,59): error TS2345: Argument of type 'null' is not assignable to parameter of type 'EventListener | EventListenerObject | undefined'.


10   (window as EventTarget).removeEventListener('testPassive', null, opts);
                                                                ~~~~

src/utils/featureDetection.ts(10,62): error TS2345: Argument of type 'null' is not assignable to parameter of type 'EventListener | EventListenerObject | undefined'.

```

It seems that we are passing `null` as the 2nd argument, where we should have passed `undefined` as per the type definition in `lib.es6.d.ts`:

```javascript
interface EventTarget {
    addEventListener(type: string, listener?: EventListenerOrEventListenerObject, options?: boolean | AddEventListenerOptions): void;
    dispatchEvent(evt: Event): boolean;
    removeEventListener(type: string, listener?: EventListenerOrEventListenerObject, options?: boolean | EventListenerOptions): void;
}
```

It looks like this got merged in #45 by @vhfmag (please chip in if you can), but seeing as this makes it impossible to run the `build`command, it seems strange that this wasn't caught, so I'm tempted to believe that this is only a problem on my end, however I cannot figure out why?
I've tried both with the bundled TS version `2.5.0` and the currently newest `2.6.1`.

Am I doing something wrong, or do we indeed need to fix this? If so, I'll happily make a PR.

**Comment by jaredpalmer** [comment]
I’ve been merging stuff without building because I took this weekend / thanksgiving off. Haven’t cut a release and don’t plan to until I check work and fix stuff.

**Comment by JReinhold** [comment]
okay, that makes total sense.
How would you feel if we added something like CircleCI integration to this repo, to test all PRs?

**Comment by jaredpalmer** [comment]
Sure I’ll add my account later tonight

**Comment by vhfmag** [comment]
My bad. I had included a fix for it in #52, but there's an isolated PR just for the fix: #56.
